queries.sql

```diff
@@ -1,3 +1,35 @@
 /*Queries that provide answers to the questions from all projects.*/
 
 SELECT * from animals WHERE name = 'Luna';
+
+/* Find all animals whose name ends in "mon". */
+
+SELECT * from animals WHERE name LIKE '%mon%';
```

Comment:
Please add a query to find all animals that have names ending with "mon" not starting and ending with "mon".